src/test/rbd_mirror/test_ClusterWatcher.cc

```diff
@@ -212,26 +212,6 @@ TEST_F(TestClusterWatcher, ReplicatedPools) {
   check_peers();
 }
 
-TEST_F(TestClusterWatcher, CachePools) {
-  PeerSpec site1("", "site1", "mirror1");
-  string base1, base2, cache1, cache2;
-  create_pool(true, site1, &site1.uuid, &base1);
-  check_peers();
-
-  create_cache_pool(base1, &cache1);
```

Comment:
You should be able to drop `create_cache_pool()` and `remove_cache_pool()` methods in the file as well.